src/app.py

```diff
@@ -2294,6 +2329,16 @@ def require_json(request):
     if not request.is_json:
         bad_request_error("A json body and appropriate Content-Type header are required")
 
+"""
+Check if the token from user request is valid
+
+token : str or Response
+    The token string if valid or Flask Response object if invalid
+"""
+def require_token(token):
```

Comment:
By doing this you lose the information returned in that response.  If you return the response then the custom message set within and specific code (400 vs 401 vs 403, but maybe sometimes 500) is sent back to the requester.

Reply:
Thanks @shirey , that's a good point. I'll make the changes.